### PR TITLE
List recent deployments as a table, for filtering

### PR DESCRIPTION
**Pull request by dsingleton**
Using the table filter feature we get free filtering by env, release number
and even by rough date. This is useful for tracking graph/error change
back to a release around a particular time.

Longer term it would be nice to surface all historic releases, and
make the release links more useful, eg permalinks for a release by id, as
a release could be deployed many times, and exposing exactly what
was deployed at that time, based on the previous release.

| Before | After |
|---|---|
| release-app-before | release-app-after |

### CODE REVIEW

File: app/views/applications/show.html.erb [hunk 1]
```diff
@@ -118,20 +118,37 @@
...
-          <span class="label label-default"><%= deployment.environment %></span>
-        <% end %>
-        at
-        <%= human_datetime(deployment.created_at) %>
-      </li>
+      <tr>
+        <td><%= human_datetime(deployment.created_at) %></td>
+        <td>
+          <% if deployment.production? %>
+            <span class="label label-danger"><%= deployment.id %></span>
```
Comment (alext) [inline app/views/applications/show.html.erb]: Why the change from `deployment.environment` to `deployment.id`?

Comment (dsingleton) [inline app/views/applications/show.html.erb]: Debugging left in by mistake :blush:

Comment (dsingleton) [inline app/views/applications/show.html.erb]: (fixed)

**Comment by fofr** [comment]
:cherry_blossom: :heart:
